packages/github-auth/src/types.ts

```diff
@@ -0,0 +1,81 @@
+export type GithubAuthVariables = {
+  token: Token | undefined
+  'refresh-token': Token | undefined
+  'granted-scopes': string[] | undefined
+  'user-github': Partial<GithubUser> | undefined
+}
+
+export type Token = {
+  token: string,
+  expires_in: number
+}
+
+export type GithubScope =
+  'repo' | 'repo:status' | 'repo_deployment' | 'public_repo' | 'repo:invite' | 'security_events' | 
+  'admin:repo_hook' | 'write:repo_hook' | 'read:repo_hook' | 'admin:org' | 'write:org' | 'read:org' |
+  'admin:public_key' | 'write:public_key' | 'read:public_key' | 'admin:org_hook' | 'gist' |
+  'notifications' | 'user' | 'read:user' | 'user:email' | 'user:follow' | 'project' | 'read:project' |
+  'delete_repo' | 'write:packages' | 'read:packages' | 'delete:packages' | 'admin:gpg_key' |
+  'write:gpg_key' | 'read:gpg_key' | 'codespace' | 'workflow'
+
+export type GithubErrorResponse = {
+  error: string
+  message: string
+  documentation_url: string
+}
+
+export type GithubTokenResponse = {
+  access_token: string
+  expires_in: number
```

Comment:
This field is omitted for OAuth Apps (but not GitHub Apps) since access tokens don't expire

Reply:
My bad, thanks!